tests/h/views/api_flags_test.py

```diff
@@ -86,6 +86,31 @@ def fake_has_permission(action, _):
 
         assert 'cannot find annotation' in exc.value.message
 
+    def test_passes_info_to_flag_notification_email(self,
+                                                    pyramid_request,
+                                                    group_service,
+                                                    flag_notification_email,
+                                                    incontext_link):
+        pyramid_request.json_body = {'annotation': annotation.id}
+        context = FakeAnnotationResourceFactory(annotation)
```

Comment:
Hmm, I'm guessing the `annotation` parameter was missing from this method, and now hound doesn't resolve this comment, right?